src/driver/driver_api.cc

```diff
@@ -252,6 +255,10 @@ Array<tvm::transform::Pass> CreatePassList(bool disable_loop_partition) {
   pass_list.push_back(
       tir::transform::CommonSubexprElimTIR(!disable_cse_tir, enable_equiv_terms_in_cse_tir));
 
+  if (instrument_lwp) {
```

Comment:
Can you add a comment that this pass needs to be last so that none of the loops get changed or re-ordered (I assume that is the reason why this pass is last).

Reply:
That's correct. Thanks for the suggestion!